### PR TITLE
highlight existence of formatting tool for contributors

### LINKED ISSUE #901: formatting tool for contributors
Do we have a formatting tool that is correctly configured so that it automatically corrects and unifies the formatting (especially indentation)? If so, where can I find it and how to use it? And would it make sense to add this reference then to the README?

### PR DESCRIPTION
**Pull request by p-se**
Although I eventually found a reference to `daps-xmlformat` documented in the [Documentation Style Guide (at the end of chapter 8)](https:/documentation.suse.comstylecurrent), where the `README.adoc` is also pointing to, I can't help but wondering if it wouldn't be a little bit better to explicitly and briefly mention its existence directly in the `README.adoc`.

What do you think?

Fixes #901 

Signed-off-by: Patrick Seidensal <pseidensal@suse.com>

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -51,6 +51,7 @@ Thank you for contributing to this repo. When creating a pull request, please fo
 
 . For help on style and structure, refer to the https://documentation.suse.com/style/current[Documentation Style Guide].
 
+. If you want to have the documentation reformatted automatically, you can either use `daps-xmlformat` for single files or `daps xmlformat` for whole documents.
```
Comment (sknorr) [inline README.adoc:54]: Sounds like a good idea.
* I tried to shorten the sentence.
* `daps xmlformat` actually does both.

```suggestion
. To reformat the documentation automatically, you can use `daps xmlformat`.
For more information, see https://opensuse.github.io/daps/doc/cha.daps.user.output.html#sec.daps.user.output.xmlformat.
```

**Review by lproven — APPROVED** [review]

File: README.adoc (hunk 1 above)
Comment (p-se) [reply README.adoc]: Note that the style guide states both. That's where I took the information from. Not sure if you want to have that updated, too.

**Review by lproven — APPROVED** [review]

**Review by lproven — APPROVED** [review]

**Review by lproven — APPROVED** [review]

**Review by tbazant — APPROVED** [review]
looks good

File: README.adoc [hunk 2]
```diff
@@ -51,6 +51,7 @@ Thank you for contributing to this repo. When creating a pull request, please fo
 
 . For help on style and structure, refer to the https://documentation.suse.com/style/current[Documentation Style Guide].
 
+. To reformat the documentation automatically, use `daps xmlformat`.
```
Comment (tbazant) [inline README.adoc:54]:
```suggestion
. Modified docbook files are re-formatted automatically on each pull request. if you still want to reformat the documentation manually, use `daps xmlformat`.
```

File: README.adoc [hunk 3]
```diff
@@ -51,6 +51,7 @@ Thank you for contributing to this repo. When creating a pull request, please fo
 
 . For help on style and structure, refer to the https://documentation.suse.com/style/current[Documentation Style Guide].
 
+. Modified docbook files are re-formatted automatically on each pull request. if you still want to reformat the documentation manually, use `daps xmlformat`.
```
Comment (tbazant) [inline README.adoc:54]:
```suggestion
. Modified DocBook files are re-formatted automatically on each pull request. If you still want to reformat the documentation manually, use `daps xmlformat`.
```

**Pull request merged**